src/types/timezone.jl

```diff
@@ -116,9 +148,12 @@ function istimezone(str::AbstractString, mask::Class=Class(:DEFAULT))
     end
 
     # Checks against pre-compiled time zones
-    tz, class = get(_TZ_CACHE, str) do
-        nothing, Class(:NONE)
+    tz_class = get(get_tz_cache(), str, nothing)
+    tz_class === nothing && return false
+    if tz_class === nothing
+        return false
+    else
+        _, class = tz_class
+        return mask & class != Class(:NONE)
     end
```

Comment:
Some redundant returns
```suggestion
    tz_class === nothing && return false
    tz, class = tz_class
    return mask & class != Class(:NONE)
```